docs/development.md

```diff
@@ -0,0 +1,133 @@
+# Information for EXTPAR Developers
+
+## Git and Github
+
+The EXTPAR code is developed using the Git version control system and the [Github web interface :material-open-in-new:](https://github.com/C2SM/EXTPAR){:target="_blank"}. 
+Outstanding bugs and requested features are tracked using the [Issues :material-open-in-new:](https://github.com/C2SM/EXTPAR/issues){:target="_blank"} section of the Github repository.  Additionally, automated testing of newly developed features is integrated into the Github interface using the Jenkins CI tool.  
+
+### Main branches
+
+The **master** branch only contains code which are released versions. 
+All commits on the master branch are tagged (`git tag -a vX.Y.Z`).
+Only the core development team is allowed to modify the master branch.
+
+### Supporting branches
+
+Any new code development should be done in a **topic** branch. Topic branches are merged
+back into master by opening a pull request. Code must be peer reviewed by the
+source code administrator.
+
+Supporting branches are removed once successfully merged in one of the master branch.
+
+### Developments with new input data sets
+
+Any new EXTPAR code that is accompanied by a new input data file or files should be added with a simultaneous pull request
+in both this code repository (for the code changes) and in the [extpar-input-data repository :material-open-in-new:](https://gitlab.dkrz.de/extpar-data/extpar-input-data){:target="_blank"}
+(for the addition of the input data files). The **topic** branch for both pull requests should have the same name, in order to enable
+the synchronization of the code and input-data repositories.
+
+### Testing new developments
+
+Once a developer has finished developing a new feature or bug fix, they should make a 
+pull request on the Github repository from their topic branch into the rc-branch.  
```

Comment:
This is not true. Should we enforce this procedure? Otherwise, we should write here that PRs should be for master branch.